dev-games/openscenegraph-openmw/openscenegraph-openmw-3.6_p20210610.ebuild

```diff
@@ -0,0 +1,162 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1,2} )
+
+WX_GTK_VER="3.0-gtk3"
+inherit cmake flag-o-matic lua-single wxwidgets
+
+DESCRIPTION="OpenMW-specific fork of OpenSceneGraph"
+HOMEPAGE="https://github.com/OpenMW/osg"
+MY_COMMIT="b02abe200c4847e73b887b064a89ea1758a5b733"
```

Comment:
Just style but so variables-to-check/edit-on-bump aren't buried, I usually do MY_ stuff like:
```
inherit ...

MY_COMMIT=
MY_P=
...

DESCRIPTION=
```
Still not all the way at the top so it's not pre-inherit pollution.